pennylane/math/quantum.py

```diff
@@ -1516,3 +1516,63 @@ def trace_distance(state0, state1, check_state=False, c_dtype="complex128"):
     eigvals = math.abs(math.eigvalsh(state0 - state1))
 
     return math.sum(eigvals, axis=-1) / 2
+
+
+def _check_trace_preserving(Ks):
+    r""" "
+    Check whether a set of Kraus operators ``Ks`` fulfills :math:`\sum_j K_j^\dagger K_j = \mathbb{1}`.
+    """
+    return np.allclose(np.sum([K.conj().T @ K for K in Ks], axis=0), np.eye(len(Ks[0])))
+
+
+def choi_matrix(Ks, check_Ks=False):
+    r"""
+    Compute the choi matrix :math:`\Lambda` of a quantum channel :math:`\mathcal{E}`
+
+    .. math:: \Lambda = (\mathbb{1} \otimes \mathcal{E})(|\phi^+ \rangle \langle \phi^+|) = \frac{1}{d} \sum_{ij=0}^{d-1} |i \rangle \langle j| \otimes \mathcal{E}(|i \rangle \langle j|)
```

Comment:
I'd replace `d` by `2^n`, because we'll never have another case, right? At least the dims of the Kraus ops are 2**n...